README.md

```diff
@@ -1,14 +1,17 @@
 ITKDoxygen
 ==========
 
-[![][doxygen-img]][doxygen-link] [![][deployment-img]][deployment-link]
+[![][doxygen-img]][doxygen-link] [![][deployment-img]][deployment-link] [![][build-docker-img]][build-docker-link]
 
 [doxygen-img]: https://github.com/InsightSoftwareConsortium/ITKDoxygen/actions/workflows/Doxygen.yml/badge.svg
 [doxygen-link]: https://github.com/InsightSoftwareConsortium/ITKDoxygen/actions/workflows/Doxygen.yml
 
 [deployment-img]: https://github.com/InsightSoftwareConsortium/ITKDoxygen/actions/workflows/pages/pages-build-deployment/badge.svg
 [deployment-link]: https://insightsoftwareconsortium.github.io/ITKDoxygen/
 
+[build-docker-img]: https://github.com/InsightSoftwareConsortium/ITKDoxygen/actions/workflows/build-docker-image.yml/badge.svg
+[build-docker-link]: https://github.com/InsightSoftwareConsortium/ITKDoxygen/actions/workflows/build-docker-image.yml
+
```

Comment:
Why are all the above changes included in this PR?

These were added in
https://github.com/InsightSoftwareConsortium/ITKDoxygen/pull/6/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
I believe that I incorrectly added a `drop` in front of that commit instead of removing it from the rebase, which is why the changes in that commit were "dropped". Because those lines were gone, I had to manually add them within the VSCode Merge Editor.

> When solving merge conflicts in the README file, part of https://github.com/InsightSoftwareConsortium/ITKDoxygen/pull/6 (I suspect it is an issue with VSCode Merge Editor) didn't show up

I'm fairly certain the issue didn't lie within VSCode Merge Editor but rather my own faulty rebase.

Reply:
:+1: Resolving.